packages/contracts/foundry.toml

```diff
@@ -3,7 +3,7 @@ src = "src"
 out = "out"
 libs = ["lib"]
 fs_permissions = [{ access = "read-write", path = "./" }]
-solc_version="0.8.23"
+solc_version="0.8.26"
```

Comment:
hi @ChaiSomsri96 - i don't think we want to bump this across our entire repo.   let's discuss.